Created RESULTS/lablog files where needed to include multiqc_report.html

victor5lm opened the pull request:
<!--
# bu-isciii tools pull request

Based on nf-core/viralrecon pull request template

Fill in the appropriate checklist below and delete whatever is not relevant.

PRs should be made against the develop of hotfix branch, unless you're preparing a software release.
-->

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] Make sure your code lints (`black and flake8`).
- If a new tamplate was added make sure:
    - [ ] Template's schema is added in `templates/services.json`.
    - [ ] Template's pipeline's documentation in `assets/reports/md/template.md` is added.
    - [ ] Results Documentation in `assets/reports/results/template.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).
- [ ] If you know a new user was added to the SFTP, make sure you added it to `templates/sftp_user.json`

## PR description

**This PR addresses #252**.

Shettland commented on bu_isciii/templates/lowfreq_panel/RESULTS/lablog_lowfreq_panel_results:
```diff
@@ -0,0 +1,9 @@
+cd $DELIVERY_FOLDER
+
+# Links to reports
+ln -s ../../../ANALYSIS/*RBPANEL/99-stats/multiqc_report.html .
```
I think there's too many backsteps here. If you are only going 1 level deep, the symbolic link should point to `../../ANALYSIS/*RBPANEL/99-stats/multiqc_report.html`. Same goes for the other lablogs in this PR

victor5lm replied:
Fixed

saramonzon commented:
Every service template has a lablog_result that includes multiqc html link? This pr only includes 4, I thought there where more lablogs that needed the change

victor5lm commented:
I fixed the paths for the symlinks and added a new line in the viralrecon results' lablog, so that a symlink to multiqc_report.html is created in RESULTS for each reference and each host.

Shettland commented on bu_isciii/templates/viralrecon/RESULTS/lablog_viralrecon_results:
```diff
@@ -20,6 +20,7 @@ cat references.tmp | while read in; do cd excel_files_${in}; ln -s ../../../ANAL
 #Create symbolic links to files that are going to be converted to excel
 cat references.tmp | while read in; do ln -s ../../ANALYSIS/*/*${in}*/variants/ivar/variants_long_table.csv ${in}_variants_long_table.csv; done
 
+for ref in $(cut -f2 ../../ANALYSIS/samples_ref.txt | sort | uniq); do for organism in $(cut -f3 ../../ANALYSIS/samples_ref.txt | tr '[:lower:]' '[:upper:]' | sort | uniq); do report="../../ANALYSIS/*_${organism}/${ref}_*_viralrecon_mapping/multiqc/multiqc_report.html"; if [ -e $report ]; then ln -s ${report} ./multiqc_report_${ref}_${organism}.html; fi; done; done
```
I would add a warning message when no multiqc is found for a certain reference/organism, 

Something like:
`$report ]; then ln -s ${report} ./multiqc_report_${ref}_${organism}.html; else echo "Multiqc report not found for reference $reference and organism $organism"; fi; done; done`
